reader_test.go

```diff
@@ -13,50 +13,74 @@ type fileSpec struct {
 	compression Compression
 	codec       CompressionCodec
 	classname   string
+	params      map[string]string
 }
 
+var gzipParams = map[string]string{GzipLevel: "1"}
+
 var files = []fileSpec{
 	{
 		"testdata/uncompressed.sequencefile",
 		NoCompression,
 		0,
 		"",
+		 nil,
+	},
+	{
+		"testdata/record_compressed_gzip.sequencefile",
+		RecordCompression,
+		GzipCompression,
+		GzipClassName,
+		nil,
```

Comment:
I didn't add this one, I added the one below it, to test with GzipLevel defined, it's just git doing its diff and chooses to show the change as adding of this one instead of the one below it.